compiler/src/iree/compiler/InputConversion/StableHLO/Passes.cpp

```diff
@@ -46,9 +46,9 @@ void buildStableHLOInputConversionPassPipelineImpl(
       stablehlo::createLegalizeControlFlow());
 
   passManager.addPass(createFlattenTuplesInSCF());
-  if (detuple) {
-    passManager.addPass(createFlattenTuplesInCFG());
-  }
+  // if (detuple) {
+  passManager.addPass(createFlattenTuplesInCFG());
+  // }
```

Comment:
?

Reply:
Blah I'll fix that. I am lazy about finding the right stablehlo input conversion.